Bottom: 29952f3d85e801b645486d52e9d0599e5014bd9c
Top:    b6fa75e2491eebe5991f2550dd1d39dcda525623
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-01-31 22:19:36 +0000

WAS: Use motion hints when processing pointer motion over the drawing area

Not sure what the remaining changes here are for!


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index e8dd61a..b4aac54 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -750,7 +750,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_LINES);
 
-#if 1
+#if 0
   if (x_prev >= 0)
     draw_crosshair (x_prev, y_prev);
 #endif
@@ -763,7 +763,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_QUADS);
 
-#if 1
+#if 0
   if (x_prev >= 0 && draw_markers_prev)
     {
       glVertex2i (0,                  y_prev - VCD);
